### PR TITLE
Properly expose offset changing at runtime.

### PR DESCRIPTION
**Pull request by keveleigh**

### CODE REVIEW

File: Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverHandler.cs [hunk 1]
```diff
@@ -217,6 +217,11 @@ private bool RequiresOffset()
             return AdditionalOffset.sqrMagnitude != 0 || AdditionalRotation.sqrMagnitude != 0;
         }
 
+        public void UpdateOffsetTransform()
```
Comment (Ecnassianer) [inline Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverHandler.cs]: This feels like it should be private

File: Assets/HoloToolkit/Utilities/Scripts/Editor/SolverHandlerEditor.cs [hunk 2]
```diff
@@ -40,21 +40,26 @@ public override void OnInspectorGUI()
...
+
             if (trackedObjectChanged)
             {
                 solverHandler.TransformTarget = null;
                 solverHandler.AttachToNewTrackedObject();
             }
+
+            if (additionalOffsetChanged)
+            {
+                solverHandler.UpdateOffsetTransform();
```
Comment (Ecnassianer) [inline Assets/HoloToolkit/Utilities/Scripts/Editor/SolverHandlerEditor.cs]: You don't need a function call here, you can just use
solverHandler.AdditionalRotation and the accessor will handle it

File: Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverHandler.cs (hunk 1 above)
Comment (keveleigh) [reply Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverHandler.cs]: I agree, but, if it's private, we can't call it from the custom editor script. Not sure if there's a better way to do this.

File: Assets/HoloToolkit/Utilities/Scripts/Editor/SolverHandlerEditor.cs (hunk 2 above)
Comment (Ecnassianer) [reply Assets/HoloToolkit/Utilities/Scripts/Editor/SolverHandlerEditor.cs]: You can add one of these:
https://docs.unity3d.com/ScriptReference/EditorGUILayout.Vector3Field.html

**Pull request merged**